### PR TITLE
My Theme

### PR DESCRIPTION
**Pull request by alecsloan**
My basic theme. Decided not to make different color scheme's since all that would change was the header and button colors.

### CODE REVIEW

File: alecs-theme/src/templates/navigation.ftl [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	<nav class="navbar-nav site-navigation navbar-right" id="navigation" data-spy="affix" data-offset-top="0" role="navigation">
+		<#assign preferencesMap = {"displayDepth": "1", "portletSetupPortletDecoratorId": "barebone"} />
+
+		<@liferay.navigation_menu
+			default_preferences=freeMarkerPortletPreferences.getPreferences(preferencesMap)
+			instance_id="main_navigation_menu"
+		/>
+
+		<div class="user-area">
+		</div>
```
Comment (jonmak08) [inline alecs-theme/src/templates/navigation.ftl]: Empty ```<div />``` can be closed on the same line.

File: alecs-theme/src/templates/portal-ext.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+combo.check.timestamp=true
+
+minifier.enabled=false
+
+module.framework.properties.osgi.console=localhost:11311
+
+com.liferay.portal.servlet.filters.cache.CacheFilter=false
+com.liferay.portal.servlet.filters.etag.ETagFilter=false
+com.liferay.portal.servlet.filters.header.HeaderFilter=false
+com.liferay.portal.servlet.filters.themepreview.ThemePreviewFilter=true
```
Comment (jonmak08) [inline alecs-theme/src/templates/portal-ext.properties]: I think this file does not need to be added?

File: alecs-theme/src/templates/portal_normal.ftl [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+						<#include "${full_templates_path}/navigation.ftl" />
+					</#if>
+				</div>
+			</header>
+
+				<div class="jumbotron jumbotron-fluid alecs-jumbotron">
+					<#if show_page_title>
+						<h1>${the_title}</h1>
+					</#if>
+				</div>
```
Comment (jonmak08) [inline alecs-theme/src/templates/portal_normal.ftl]: Fix indentation

File: alecs-theme/src/css/partials/_header.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	.navbar-user {
+		.user-avatar-link, .sign-in {
+			& > a {
+				color: $main-header-color;
+				display: inline-block;
+				font-size: 12px;
+				padding: 16px 16px 16px 6px;
+				text-transform: uppercase;
+
+				&:hover, &:focus, &.active, &.open {
```
[the remark this answers is not in the document]
Comment (alecsloan) [reply alecs-theme/src/css/partials/_header.scss]: Not sure if I got this right in the upcoming pull

File: alecs-theme/src/templates/footer.ftl [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+					default_preferences=freeMarkerPortletPreferences.getPreferences(preferencesMap)
+					instance_id="footer_navigation_menu"
+				/>
+			</div>
+		</div>
+	</nav>
+	<#include "${full_templates_path}/social_media.ftl" />
+	<div class="outro">
+		<hr />
+		${company_name} <p id="copyright"><small><@liferay.language key="powered-by" /> <a href="http://www.liferay.com" rel="external">Liferay</a></small></p>
```
[the remark this answers is not in the document]
Comment (alecsloan) [reply alecs-theme/src/templates/footer.ftl]: ${company_name} will change. But "powered by Liferay" and link will remain to show they're using a Liferay portal.

**Pull request closed without merging**